oeo-physical.omn

```diff
@@ -3243,7 +3257,8 @@ Class: Wind
         <http://purl.obolibrary.org/obo/IAO_0000119> "REGULATION (EC) No 1099/2008 OF THE EUROPEAN PARLIAMENT AND OF THE COUNCIL of 22 October 2008 on energy statistics https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32008R1099&from=EN"
     
     SubClassOf: 
-        PortionOfAir
+        PortionOfAir,
```

Comment:
For consistency reasons, it would be nicer to speak of either 'WindPower' and 'SolarPower' or else leave '...power' out. For both, we should discuss the classification. Does it make sense, that they are classified differently? Looking at the big picture, wind is also induced by solarPower.

Reply:
I'll change wind to windpower for now and we should make an issue about that because it seems to be a bit complicated